src/components/MakeWidget/MakeWidget.tsx

```diff
@@ -310,6 +314,18 @@ const MakeWidget: FC = () => {
     }
   };
 
+  const handleTakerAddress = async (value: string) => {
```

Comment:
Same here, i think we should use a method return type.

Reply:
we no longer need this. I'm removing it